cla_backend/apps/core/fixtures/test_auth_clients.json

```diff
@@ -84,6 +97,24 @@
         }
     },
     {
+        "pk": 95, 
```

Comment:
this fixture doesn't get applied on prod - the API client and user will get manually created in prod instead.